src/main/java/com/codingnomads/betty/logic/models/SentimentResult.java

```diff
@@ -0,0 +1,41 @@
+package com.codingnomads.betty.logic.models;
+
+public class SentimentResult {
```

Comment:
You can just delete this POJO and use the TeamSentimentScore one. Remember, you also need to pass other information from the tweet / match as per the fields

Reply:
Same as my comment above.